src/main/java/org/datadog/jenkins/plugins/datadog/listeners/DatadogBuildListener.java

```diff
@@ -315,25 +315,24 @@ public void onCompleted(Run run, @Nonnull TaskListener listener) {
             // Send a metric
             Map<String, Set<String>> tags = buildData.getTags();
             String hostname = buildData.getHostname(DatadogUtilities.getHostname(null));
-            metrics.gauge("jenkins.job.duration", buildData.getDuration(0L) / 1000, hostname, tags);
+            metrics.gauge("jenkins.job.duration", TimeUnit.MILLISECONDS.toSeconds(buildData.getDuration(0L)), hostname, tags);
             logger.fine(String.format("[%s]: Duration: %s", buildData.getJobName(), toTimeString(buildData.getDuration(0L))));
 
             if (run instanceof WorkflowRun) {
                 RunExt extRun = getRunExtForRun((WorkflowRun) run);
-                long pauseDuration = 0;
+                long pauseDurationMillis = 0;
                 for (StageNodeExt stage : extRun.getStages()) {
-                    pauseDuration += stage.getPauseDurationMillis();
+                    pauseDurationMillis += stage.getPauseDurationMillis();
                 }
-                metrics.gauge("jenkins.job.pause_duration", pauseDuration / 1000, hostname, tags);
-                logger.fine(String.format("[%s]: Pause Duration: %s", buildData.getJobName(), toTimeString(pauseDuration)));
-                long buildDuration = run.getDuration() - pauseDuration;
-                metrics.gauge("jenkins.job.build_duration", buildDuration / 1000, hostname, tags);
+                metrics.gauge("jenkins.job.pause_duration", TimeUnit.MILLISECONDS.toSeconds(pauseDurationMillis), hostname, tags);
+                logger.fine(String.format("[%s]: Pause Duration: %s", buildData.getJobName(), toTimeString(pauseDurationMillis)));
+                long buildDuration = run.getDuration() - pauseDurationMillis;
```

Comment:
```suggestion
                long buildDurationMillis = run.getDuration() - pauseDurationMillis;
```
At some point, probably we want to start using `java.time.Duration` and `java.time.Instant` to get rid of these `xxxMillis` suffixes everywhere, now that `JDK 1.8` is a requirement for the plugin. (Not blocker for this PR, tho)

Reply:
Agree about the `java.time` usage. Added the suffix for now :)